package.json

```diff
@@ -28,6 +28,8 @@
     "build:mac": "yarn run compile && electron-builder --mac --dir",
     "build:win": "yarn run compile && electron-builder --win --dir",
     "integration": "jest --runInBand --detectOpenHandles --forceExit integration",
+    "test:unit": "jest --watch --testPathIgnorePatterns integration",
```

Comment:
I discovered you need to make sure you have an up-to-date build before running this

Reply:
Yes, this is intentional. The script is for development, and for that, it is optimized for quick start-up.